analyze_issue_metadata.py

```diff
@@ -0,0 +1,186 @@
+import csv
+import json
+import os
+import sys
+
+from datetime import datetime as dt
+
+import github3
+
+import get_issue_metadata as gim
+
+
+def main(state):
+
+    """
+    Command line tool that reads in an input file
+    containing GitHub repository names, calls read_issue_metadata
+    and returns a JSON file of issue data.
+
+    Usage (command line):
+        $ python analyze_issue_metadata.py googlecloudplatform repos.txt
+
+    Args:
+        state: GitHub state of issue ('open', 'closed'
+               or 'all')
+        sys.argv[1]: github owner or organization
```

Comment:
These are not technically arguments to this method, but you should make them actual arguments. Make them parameters on `main` and pass them in your `if __name__ == '__main__'` clause at the bottom.